packages/@okta/vuepress-site/docs/guides/sampleapp-oie-redirectauth/main/index.md

```diff
@@ -128,37 +130,55 @@ You can modify the Application's Sign-On Policy to require the user to have a se
 
 2. From the side navigation, select **Applications** > **Applications** and then select the Okta OAuth app that you created to represent the <StackSnippet snippet="applang" inline /> app.
 3. Select the **Sign On** tab.
-4. Scroll down to the **Sign On Policy** section, click the **Actions** menu icon (&#8942;) beside the **ENABLED** flag and select **Edit**.
-5. In the Edit Rule dialog box, scroll down to the **THEN** section and locate **AND User must authenticate with**.
-6. Select **Password + Another factor** and click **Save**.
+4. Scroll down to the **User authentication** section and click **Edit**.
+5. From the **Authentication policy** dropdown menu, select **Any two factors** and click **Save**.
 
 ### Try multifactor authentication
 
 <StackSnippet snippet="tryenrollin" />
 
-3. Enter the credentials of the user that you enrolled earlier. The Set up authentications page appears, which prompts you to set up either the Okta Verify or the Phone authenticator.
+3. Enter the credentials of the user that you enrolled earlier. The Set up security methods page appears, which prompts you to set up either the Okta Verify app or the Phone authenticator.
```

Comment:
Consider rewording to: "Enter the credentials of the user that you enrolled with earlier. The Set up security methods page appears, prompting you to set up either the Okta Verify app or the Phone authenticator."